06_puppy/patrickmarabeas/syncStore_test.go

```diff
@@ -0,0 +1,95 @@
+package main
```

Comment:
Is this a duplicate of `mapStore_test.go` with a different backend? You should implement a single set of parameterised tests.

Reply:
Tests have been refactored.